db/migrate/20240122064113_create_ad_campaigns.rb

```diff
@@ -0,0 +1,35 @@
+class CreateAdCampaigns < ActiveRecord::Migration[7.1]
+  def change
+    create_table :ad_campaigns, :id => false, primary_key: :ad_campaign_id do |t| 
```

Comment:
just a fyi, we keep Rails auto generated id - that helps in referring to an object internally without relying on external id's presence. for this exercise it is fine, I am just mentioning it as an fyi

Reply:
I considered doing that. I felt that the key was unnecessary. At least in this scenario, FB returned ID is unique at each level.